src/RadioGroup/RadioGroup.tsx

```diff
@@ -46,8 +47,24 @@ export default class RadioGroup<
   _optionRefsByID = {};
 
   render() {
-    const { className, disabled, error, label, onChange, options, selectedID } = this.props;
+    const {
+      ariaLabelledBy,
+      className,
+      disabled,
+      error,
+      label,
+      onChange,
+      options,
+      selectedID,
+    } = this.props;
+
+    if (ariaLabelledBy && label) {
+      console.warn(
+        "You should not pass both `ariaLabelledBy` and `label` props. The `ariaLabelledBy` prop is ignored.",
```

Comment:
We could try to do this check at compile time with a pattern along the lines of https://gist.github.com/brian-mcallister-lab49/02c43693d5251daa6e0e7335bed950e2... but it's probably not necessary. I think at least a runtime warning is fine.

Reply:
Oh that's a neat trick, but yeah I think this should be fine as it is since the outcome of having both props passed in is not bad and still accessible